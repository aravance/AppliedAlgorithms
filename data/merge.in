chocolate chips cchocholaiptes
chocolate chips chocochilatspe
ababc babd ababdbabc
ababc babd ababcbabd
catch cabcd ccatabcdch
catch cabcd ccabatcchd
catch cacd ccatacdch
catch cbcd cbcatcdch
