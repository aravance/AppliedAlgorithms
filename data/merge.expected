CchocHolaIPteS
No merge possible!
aBABDbabc
ababcBABD
CcatABCDch
CcABatCchD
CcatACDch
CBcatCDch
